scripts/algorithms/l2-svm.dml

```diff
@@ -26,6 +26,23 @@
 # Assume input and output directories are on hdfs as INPUT_DIR and OUTPUT_DIR
 # Assume epsilon = 0.001, lambda = 1, maxiterations = 100
 #
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME      TYPE    DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X         String  ---         Location to read the matrix X of feature vectors
+# Y         String  ---         Location to read response matrix Y
+# icpt      Int     0           Intercept presence
+#                               0 = no intercept
+#                               1 = add intercept;
+# tol       Int     0.001       Tolerance (epsilon);
+# reg       int     1.0         Regularization parameter (lambda) for L2 regularization
+# maxiter   Int     100         Maximum number of conjugate gradient iterations, 0 = no maximum
+# model     Int     1           Location to write model
+# Log       String  ---         Location to write the log file
```

Comment:
@dusenberrymw do you mean `Log    Optional    ----    Location?` + move this to the end after fmt?.